### DISCUSSION TITLE
Where else might the descriptive-ness of the spec be reduced?

### DISCUSSION THREAD

**Discussion by earth2marsh** [OAI/sig-moonwalk#14] [General]
What about matrix parameters or exploding parameters? Should discriminator continue into the future? If we wanted to decrease the tooling burden, what might be some things to remove?

**Comment by darrelmiller** [comment]
Support for these capabilities will come with any standard UriTemplate library.  Tooling should simply take a dependency on an existing library.  I think introducing partial support for URI Templates would actually increase the work for tooling folks.
